hack/aks/Makefile

```diff
@@ -228,6 +228,44 @@ linux-cniv1-up: rg-up overlay-net-up ## Bring up a Linux CNIv1 cluster
 
 	@$(MAKE) set-kubeconf
 
+dualstack-overlay-up: rg-up overlay-net-up ## Brings up an dualstack Overlay cluster with Linux node only
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--kubernetes-version 1.26.3 \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--network-plugin azure \
+		--ip-families ipv4,ipv6 \
+		--network-plugin-mode overlay \
+		--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/AzureOverlayDualStackPreview \
+		--subscription $(SUB) \
+		--no-ssh-key \
+		--yes
+	@$(MAKE) set-kubeconf
+
+dualstack-overlay-byocni-up: rg-up overlay-net-up ## Brings up an dualstack Overlay BYO CNI cluster
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--kubernetes-version 1.26.3 \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--network-plugin none \
+		--network-plugin-mode overlay \
+		--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/AzureOverlayDualStackPreview \
+		--ip-families ipv4,ipv6 \
+		--subscription $(SUB) \
+		--no-ssh-key \
+		--yes
+	@$(MAKE) set-kubeconf
+
+dualstack-windows-byocni-up: ## Add windows node pool to dualstack overlay cluster
+	$(AZCLI) aks nodepool add -g $(GROUP) -n npwin \
+	    --cluster-name  $(CLUSTER) \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--os-type Windows \
+		--os-sku Windows2022 \
+		--subscription $(SUB)
+	@$(MAKE) set-kubeconf 
```

Comment:
I don't believe there is anything dualstack specific about this windows nodepool. Lets change this to `windows-nodepool-up` and parameterize it similarly to `windows-cniv1-up`.

Reply:
that's right, I will use another PR for windows